wiki/Skinning/Compendium/en.md

```diff
@@ -129,88 +156,123 @@ Compendium of all completed osu! skins.
 - [Redd Glass](https://osu.ppy.sh/community/forums/topics/211396) by [Haskorion](https://osu.ppy.sh/users/3252321)![][o!s]![][o!t]![][o!c]![][o!m]
 - [Reisen Udongein Inaba](https://osu.ppy.sh/community/forums/topics/34563) by [MLGnom](https://osu.ppy.sh/users/46620)![][o!s]
 - [Reiuji Utsuho](https://osu.ppy.sh/community/forums/topics/105440) by [MLGnom](https://osu.ppy.sh/users/46620)![][o!s]
 - [Remilia Scarlet](https://osu.ppy.sh/community/forums/topics/46112) by [MLGnom](https://osu.ppy.sh/users/46620)![][o!s]
 - [Retro Vibe](https://osu.ppy.sh/community/forums/topics/654183) by [Galvit](https://osu.ppy.sh/users/7629682)![][o!s]
 - [Re:m](https://osu.ppy.sh/community/forums/topics/481314) by [Choilicious](https://osu.ppy.sh/users/2129634)![][o!s]
+- [re:Shigetora](https://osu.ppy.sh/community/forums/topics/486160) by [_Indeketsu](https://osu.ppy.sh/users/6286572)![][o!s]![][o!t]![][o!c]
+- [Re:Zero](https://osu.ppy.sh/community/forums/topics/473335) by [Choilicious](https://osu.ppy.sh/users/2129634)![][o!s]
+- [re;Line Meet The Flat](https://osu.ppy.sh/community/forums/topics/647228) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]![][o!t]![][o!c]![][o!m]
+- [re;Minimalist Lolicon](https://osu.ppy.sh/community/forums/topics/661777) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]![][o!c]![][o!m]
+- [Re;RamRem](https://osu.ppy.sh/community/forums/topics/574042) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]![][o!m]
+- [Rising Sun](https://osu.ppy.sh/community/forums/topics/613471) by [Hostile](https://osu.ppy.sh/users/8818035)![][o!s]
+- [ro-500](https://osu.ppy.sh/community/forums/topics/651827) by [dlwnstns99](https://osu.ppy.sh/users/4585661)![][o!s]
+- [RWBY Rose](https://osu.ppy.sh/community/forums/topics/473353) by [Choilicious](https://osu.ppy.sh/users/2129634)![][o!s]
 - [RWBY 2 Black](https://osu.ppy.sh/community/forums/topics/351477) by [akkord](https://osu.ppy.sh/users/2129634)![][o!s]
+- [R4V3 F4L7](https://osu.ppy.sh/community/forums/topics/528744) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]![][o!t]![][o!c]![][o!m]
 
 ## S
 
 - [Sagiri Edition](https://osu.ppy.sh/community/forums/topics/588711) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]
+- [Sagiri](https://osu.ppy.sh/community/forums/topics/631673) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]
+- [Saikuus](https://osu.ppy.sh/community/forums/topics/620379) by [Saikkimus](https://osu.ppy.sh/users/9820272)![][o!s]![][o!t]![][o!c]![][o!m]
 - [Sakuya Izavoi](https://osu.ppy.sh/community/forums/topics/32589) by [MLGnom](https://osu.ppy.sh/users/46620)![][o!s]
+- [Sanoba Witch](https://osu.ppy.sh/community/forums/topics/654114) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]![][o!c]![][o!m]
 - [Senki Zesshou Symphogear](https://osu.ppy.sh/community/forums/topics/181715) by [Kurokoami](https://osu.ppy.sh/users/260933)![][o!s]![][o!t]![][o!c]![][o!m]
```

Comment:
While you are at it, can you fix my name and remove the extra "o" after the second k? 🤔 👀

Reply:
Oh sorry, will do that when I fix the other things